src/Pages/BookDetails/Aside.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+function Aside() {
+  const [save, setSave] = useState(false);
+
+  return (
+    <AsideContainer>
+      <Container>
+        <ul>
+          <Menu save={save} onClick={() => setSave(!save)}>
+            <img
+              alt=""
+              src={
+                save
+                  ? "https://i.ibb.co/wSDskYv/shhhh.png"
+                  : "https://cdn4.iconfinder.com/data/icons/app-custom-ui-1/48/Download-256.png"
+              }
+            />
+            <button>내 서재에 담기</button>
+          </Menu>
+          <Menu>
+            <img
+              alt=""
+              src="https://cdn4.iconfinder.com/data/icons/basic-user-interface-elements/700/share-sharing-socialmedia-social-network-exchange-256.png"
+            />
+            <button>공유하기</button>
+          </Menu>
+        </ul>
+        <ReadNow>바로 읽기</ReadNow>
+      </Container>
+    </AsideContainer>
+  );
+}
+
+export default Aside;
+
+const AsideContainer = styled.div`
+  width: 280px;
+  position: relative;
+`;
+
+const Container = styled.aside`
+  position: fixed;
+  top: 64px;
+  width: 280px;
+  height: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-direction: column;
+  border-left: 1px solid rgb(223, 223, 223);
+  border-right: 1px solid rgb(223, 223, 223);
+`;
+
+const Menu = styled.li`
+  height: 47px;
+  width: 256px;
+  background-color: transparent;
+  border-bottom: 1px solid rgb(223, 223, 223);
+  padding-left: 25px;
+  display: flex;
+  align-items: center;
+  img {
+    width: 20px;
+    height: 20px;
+  }
+  button {
+    font-size: 15px;
+    background-color: transparent;
+    font-weight: 750;
+    text-align: left;
+    line-height: 47px;
+    padding-left: 10px;
+    color: ${(props) => (props.save ? "rgb(193,193,193)" : "")};
```

Comment:
```suggestion
    color: ${({ isSaved }) => isSaved && "rgb(193,193,193)"};
```
- 함수이므로 파라미터 자리에서 구조분해 할당 가능합니다
- true일 경우에 대해서만 처리하고 싶다면 &&를 사용해주세요